src/Checkbox.js

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import { rem } from 'polished';
+
+import Box from './Box';
+import Flex from './Flex';
+import Text from './Text';
+import Icon from './Icon';
+
+const CheckIcon = ({ fill = 'currentColor', ...props }) => (
+  <svg width="10" height="7" viewBox="0 0 10 7" {...props}>
+    <path
+      fill={fill}
+      fillRule="evenodd"
+      d="M4 4.586L1.707 2.293A1 1 0 1 0 .293 3.707l3 3a.997.997 0 0 0 1.414 0l5-5A1 1 0 1 0 8.293.293L4 4.586z"
+    />
+  </svg>
+);
+
+CheckIcon.propTypes = {
+  fill: PropTypes.string.isRequired
+};
+
+const MinusIcon = ({ fill = 'currentColor', ...props }) => (
+  <svg width="16" height="16" viewBox="0 0 16 16" {...props}>
+    <path
+      fill={fill}
+      fillRule="evenodd"
+      d="M11 7H5c-.55 0-1 .45-1 1s.45 1 1 1h6c.55 0 1-.45 1-1s-.45-1-1-1z"
+    />
+  </svg>
+);
+
+MinusIcon.propTypes = {
+  fill: PropTypes.string.isRequired
+};
+
+const Label = styled(Flex.withComponent('label'))`
```

Comment:
We have a `Label` component in Arbor, is it not suitable here?

Reply:
You're completely right, they should be merged. Our current Label holds text input-specific styling decisions and I didn't want to break anything there.